schnapps/core/schnapps.cpp

```diff
@@ -514,8 +514,10 @@ void SCHNApps::cycle_selected_view()
 View* SCHNApps::split_view(const QString& name, Qt::Orientation orientation)
 {
 	View* new_view = add_view();
-
 	View* view = views_.at(name).get();
+
+	emit(view_splitted(view));
```

Comment:
Petit point conjugaison : to split est un verbe irrégulier dont le participe passé est "split".
Est-ce qu'on peut renommer ce signal en "view_split" ?.. ;-)